ZelloChannelKit/ZelloChannelKit/network/ZCCSocket.m

```diff
@@ -627,6 +627,7 @@ - (void)handleStreamStart:(NSDictionary *)encoded original:(NSString *)original
     [self reportInvalidJSONInMessage:ZCCEventOnStreamStart key:ZCCStreamCodecHeaderKey errorDescription:@"codec_header missing or invalid" original:original];
     return;
   }
+  header = [header stringByTrimmingCharactersInSet:NSCharacterSet.whitespaceAndNewlineCharacterSet];
```

Comment:
Any other calls where this logic should apply?

Reply:
I don't think so. This is the only place where we decode a base64 string from another client.